ci/do_ci.sh

```diff
@@ -39,15 +39,38 @@ function bazel_with_collection() {
   collect_build_profile $1
 }
 
+function cp_binary_for_outside_access() {
+  DELIVERY_LOCATION=$1
+  cp -f \
+    "${ENVOY_SRCDIR}"/bazel-bin/source/exe/envoy-static \
+    "${ENVOY_DELIVERY_DIR}"/${DELIVERY_LOCATION}
+}
+
+function bazel_sizeopt_binary_build() {
+  echo "Building..."
+  bazel build ${BAZEL_BUILD_OPTIONS} -c opt //source/exe:envoy-static --define optimize_binary_size=enabled
```

Comment:
Why are we passing `-c opt` here? I guess order matters? Is there a way to only pass `-Os` and not I'm guessing effectively `-O2 -Os`

Reply:
Bazel doesn't have a size optimized compilation type. We have to override whatever is there via copt.